### ISSUE TITLE
`every` does not send the latest values of the input components

### ISSUE THREAD

**Issue by abidlabs** [gradio-app/gradio#6577]
### Describe the bug

If you create an event with `every`, it only sends the original values of the input components, instead of the latest.

### Have you searched existing issues?  🔎

- [X] I have searched and found no existing issues

### Reproduction

```py
import gradio as gr

with gr.Blocks() as demo:
    c1 = gr.Checkbox()
    c2 = gr.Checkbox()
    demo.load(lambda x:x, c1, c2, every=1)
    
demo.launch()
```

### System Info

```shell
gradio==4.7.1
```

### Severity

I can work around it

**Comment by howardwu1** [comment]
I noticed the same thing. What is the work around?

**Comment by thiswillbeyourgithub** [comment]
Have had this issue for a while. This is problematic and I haven't found a workaround so far. I resort to clicking manually on a button to get a "status" value.

**Comment by redevined** [comment]
Any updates on this? I don't really see a workaround either.
```python
import gradio as gr

with gr.Blocks() as demo:
    out = gr.Textbox()

    @demo.load(outputs=out, every=10)
    def periodic_task():
        return read_rows_from_some_database()
    
demo.launch()
```

**Comment by howardwu1** [comment]
My workaround ended up requiring me to cancel the queue events whenever ui elements were interacted with. Not elegant but gets the job done with several lines of code if you have multiple ui.

**Comment by freddyaboulton** [comment]
@redevined Can you explain what your problem is? It seems different from what this issue is tackling as your demo does not have any input components.

@howardwu1 - that's actually the correct approach here. It's shown in the `live_dashboard` demo:  https://github.com/gradio-app/gradio/blob/6be3c2c47a616c904c8497d1fbef7a851c54d488/demo/live_dashboard/run.py#L55 

It's a bit wordy but the alternative would require gradio to launch a different server request for each iteration which I don't think would scale well. Going to think about this some more.

**Comment by redevined** [comment]
Sorry for the confusion, seems like I simplified my example a bit too much.
